cmd/oras/pull.go

```diff
@@ -122,12 +123,21 @@ func runPull(opts pullOptions) error {
 					s.Annotations = make(map[string]string)
 				}
 				s.Annotations[ocispec.AnnotationTitle] = configPath
```

Comment:
What will happen if `configPath` is empty?

Reply:
Then `s` becomes unnamed.